Sources/ForageSDK/Foundation/Protocol/ForagePANElement.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ForagePANElement.swift
+//
+//
+//  Created by Danny Leiser on 11/23/23.
+//  Copyright © 2023-Present Forage Technology Corporation. All rights reserved.
+//
+
+/// Represents card information derived from the user's current Primary Account Number (PAN) input value.
+public protocol CardInformation {
+    
+    /// The US state that issued the EBT card, derived from the Issuer Identification Number (IIN),
+    /// also known as BIN (Bank Identification Number).
+    /// The IIN is the first 6 digits of the PAN.
+    var usState: USState? { get }
```

Comment:
I touched on this in one of the existing threads as well but I wonder if a simpler initial modification could be only exposing `iin` instead of `usState` since `iin` could apply beyond just EBT cards and `iin` technically communicates the US State if you have access to the list of US State IINs. We could always add a `usState` field later if we want.